docs/CONTRIBUTING.md

```diff
@@ -13,56 +16,50 @@ The sections below outline the steps in each case.
+In case you feel like you've made a valuable contribution, but you don't know
+how to write or run tests for it, or how to generate the documentation: don't
+let this discourage you from making the pull request; we can help you! Just go
+ahead and submit an issue and ask your questions.
 
 ## You want to make a new release of the code base
 
 To create a release you need write permission on the repository.
 
-1. Check the author list in [`CITATION.cff`](CITATION.cff)
-1. Bump the version using `bump-my-version bump <major|minor|patch>`. For example, `bump-my-version bump major` will increase major version numbers everywhere it's needed (code, meta, etc.) in the repo. Alternatively the version can be manually changed in diffwofost/__init__.py, pyproject.toml, CITATION.cffand docs/conf.py (and other places it was possibly added).
-
-1. Go to the [GitHub release page](https://github.com/WUR-AI/diffwofost/releases)
-1. Press draft a new release button
-1. Fill version, title and description field
-1. Press the Publish Release button
-<!--
-For projects that automatically publish to PyPI using a release or publish workflow, something like the following could be useful to add (make sure to replace the names and links):
-
-1. Wait until [PyPi publish workflow](https://github.com/WUR-AI/diffwofost/actions/workflows/publish.yml) has completed
-1. Verify new release is on [PyPi](https://pypi.org/project/matchms/#history)
--->
-<!--
-For projects that also build conda packages, e.g. on conda-forge or Bioconda, something like the following could be useful to add (example taken from matchms, make sure to replace the names and links):
-
-1. Wait until new release is also on Bioconda (https://anaconda.org/bioconda/diffwofost) via a automaticly created PR on [bioconda recipes repo](https://github.com/bioconda/bioconda-recipes/pulls?q=is%3Apr+is%3Aopen+diffwofost)
-1. Test diffwofost from bioconda by manually running [Conda verify](https://github.com/WUR-AI/diffwofost/actions/workflows/conda_verify.yml) workflow
--->
-
-Also a Zenodo entry will be made for the release with its own DOI.
+1. Check the author list in `CITATION.cff`in
+   the root of the repository.
+2. Bump the version. The version can be manually changed in `pyproject.toml` in
+   the root of the repository. Follow [Semantic Versioning](https://semver.org/)
+   principles.
+
+3. Go to the [GitHub release
+   page](https://github.com/WUR-AI/diffwofost/releases). Press draft a new
+   release button. Fill version, title and description field. Press the Publish
+   Release button
+
+4. This software automatically publish to PyPI using a release or publish
+   workflow. Wait until [PyPi publish
+   workflow](https://github.com/WUR-AI/diffwofost/actions/workflows/python-publish.yml)
+   has completed and verify new release is on
+   [PyPi](https://pypi.org/project/matchms/#history)
```

Comment:
Is the hook that makes a snapshot on Zenodo  also activated? Maybe it could be mentioned here - or maybe not, since there is no action needed by the user.